components/automate-ui/src/app/pages/project/list/project-list.component.html

```diff
@@ -111,6 +110,13 @@
         (confirm)="confirmApplyStop()"
         (cancel)="cancelApplyStop()">
       </app-confirm-apply-stop-modal>
+
+      <app-message-modal
+        [title]="'Could Not Delete Project'"
```

Comment:
@susanev should weigh in on this, but should we change the color of the title or message so it's more clear something is wrong? Right now it looks similar enough to the regular delete modal that folks might click `close` mindlessly without reading the message.
<img width="755" alt="Screen Shot 2019-12-17 at 3 44 29 PM" src="https://user-images.githubusercontent.com/21015366/71032916-5772e000-20e4-11ea-862d-36c6447edc8b.png">
<img width="740" alt="Screen Shot 2019-12-17 at 3 44 40 PM" src="https://user-images.githubusercontent.com/21015366/71032926-59d53a00-20e4-11ea-97fa-8ef339cc4382.png">

Reply:
Confirmed with @susanev to leave as is for now. Thanks for raising the question!